bytebase/bytebase.com
# fix: fixing bugs 31 05

**This PR:**
- Change button link “Learn more” from [cal.com](http://cal.com/) to https://www.bytebase.com/request-demo/;
- Combine layouts for databases and integrations;
- Add plus to table of content section;
- Fix landing page left spacing.

**Steps to test**

- Open the [Home page]()
- Open the [Blog page]()
- Confirm that everything looks and works as expected
